tools/importer-rest-api-specs/components/parser/dataworkarounds/workarounds.go

```diff
@@ -23,6 +23,8 @@ var workarounds = []workaround{
 
 	// @tombuildsstuff: we also have to account for package names which aren't valid in Go:
 	workaroundInvalidGoPackageNames{},
+
+	workaroundMachineLearning25142{},
```

Comment:
Can we put this up the top with the other service fixes? These two at the bottom are kinda special-case catch-all's